### PR TITLE
Add missing concept blurbs

### LINKED ISSUE #1427: Add blurb to Concept Exercises
Each Concept will have to define a _blurb_, which is a short description of the exercise.

In an earlier PR, a placeholder blurb has been added to the .meta/config.json file of each Concept Exercise. The placeholder blurbs should be replaced with an actual blurb for each Concept Exercise. Blurbs must be limited to 350 chars and will be truncated in some views.

For forked exercises, it might be useful to check how other tracks have defined their blurb.

The blurb will be displayed on a track's exercises page and on exercise tooltips. For example:

Screenshot 2021-03-02 at 13 25 38
Screenshot 2021-03-02 at 13 25 51

See the [Concept Exercise spec](https://github.com/exercism/docs/blob/main/anatomy/tracks/concept-exercises.md#file-metaconfigjson) for more information.

## Tracking

https://github.com/exercism/v3-launch/issues/23

### PR DESCRIPTION
**Pull request by andrerfcsantos**
Fixes #1427

### CODE REVIEW

File: concepts/conditionals-if/.meta/config.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "blurb": "TODO: add blurb",
+  "blurb": "If statements allow to control the flow of execution of a program based on a boolean condition.",
```
Comment (jmrunkle) [inline concepts/conditionals-if/.meta/config.json:2]: Grammar improvement:

If statements control the flow of execution of a program based on a boolean condition.

File: concepts/conditionals-switch/.meta/config.json [hunk 2]
```diff
@@ -1,5 +1,5 @@
 {
-  "blurb": "TODO: add blurb",
+  "blurb": "Switches allow to control the flow of a program based on the value of a variable or expression.",
```
Comment (jmrunkle) [inline concepts/conditionals-switch/.meta/config.json:2]: Grammar improvement:

Switches control the flow of a program based on the value of a variable or expression.

File: concepts/maps/.meta/config.json [hunk 3]
```diff
@@ -1,5 +1,5 @@
 {
-  "blurb": "TODO: add blurb",
+  "blurb": "Maps are a bultin type in Go that allow to associate a piece of data with a key.",
```
Comment (jmrunkle) [inline concepts/maps/.meta/config.json:2]: Grammar improvement:

Maps are a bultin type in Go that associates values with keys.

File: concepts/methods/.meta/config.json [hunk 4]
```diff
@@ -1,5 +1,5 @@
 {
-  "blurb": "TODO: add blurb",
+  "blurb": "Methods are similar to functions, but are associated with a particular type and can be called on values of the type they are associated.",
```
Comment (jmrunkle) [inline concepts/methods/.meta/config.json:2]: Simplification:

Methods are functions with a special receiver argument that can be called on an instance of the receiver type.

File: concepts/nil/.meta/config.json [hunk 5]
```diff
@@ -1,5 +1,5 @@
 {
-  "blurb": "TODO: add blurb",
+  "blurb": "nil is a special value in Go to represent the absence of a value. It is the zero value of many types in Go.",
```
Comment (jmrunkle) [inline concepts/nil/.meta/config.json:2]: Remove the second sentence for the blurb.

File: concepts/range-iteration/.meta/config.json [hunk 6]
```diff
@@ -1,5 +1,5 @@
 {
-  "blurb": "TODO: add blurb",
+  "blurb": "Go has a special \"for range\" loop to easily iterate over collections of data.",
```
Comment (jmrunkle) [inline concepts/range-iteration/.meta/config.json:2]: I believe range is formally a function, so let's say:

Go has a special range function to easily iterate over collections of data.

File: concepts/slices/.meta/config.json [hunk 7]
```diff
@@ -1,5 +1,5 @@
 {
-  "blurb": "TODO: add blurb",
+  "blurb": "Slices are lists of items of the same type. Individual elements from slices can be accessed, removed and added.",
```
Comment (jmrunkle) [inline concepts/slices/.meta/config.json:2]: Again, remove the second sentence. The blurb is sufficient without it.

File: concepts/strings-package/.meta/config.json [hunk 8]
```diff
@@ -1,5 +1,5 @@
 {
-  "blurb": "TODO: add blurb",
+  "blurb": "The strings package is included in the standard library and includes great utility functions to work with strings.",
```
Comment (jmrunkle) [inline concepts/strings-package/.meta/config.json:2]: Simplification:

The standard library strings package includes great utility functions to work with strings.

File: concepts/structs/.meta/config.json [hunk 9]
```diff
@@ -1,5 +1,5 @@
 {
-  "blurb": "TODO: add blurb",
+  "blurb": "Structs allow to group pieces of related data together.",
```
Comment (jmrunkle) [inline concepts/structs/.meta/config.json:2]: Slightly better grammar / simplification:

Structs group pieces of related data together.

File: concepts/type-conversion/.meta/config.json [hunk 10]
```diff
@@ -1,5 +1,5 @@
 {
-  "blurb": "TODO: add blurb",
+  "blurb": "Go allows to convert values from one type to another.",
```
Comment (jmrunkle) [inline concepts/type-conversion/.meta/config.json:2]: Slightly better grammar / simplification:

Go can convert values from one type to another.

File: concepts/zero-values/.meta/config.json [hunk 11]
```diff
@@ -1,5 +1,5 @@
 {
-  "blurb": "TODO: add blurb",
+  "blurb": "When a variable is created without a value, it gets assigned the zero value of its type. Learn about the zero value of the different types in Go.",
```
Comment (jmrunkle) [inline concepts/zero-values/.meta/config.json:2]: Second sentence is unnecessary, please remove it.

**Review by jmrunkle — CHANGES_REQUESTED** [review]
Thanks for this contribution. Please address my comments then we can merge this.

File: concepts/range-iteration/.meta/config.json (hunk 6 above)
Comment (andrerfcsantos) [reply concepts/range-iteration/.meta/config.json:2]: range [is a keyword](https://golang.org/ref/spec#Keywords), so it's not accurate to call it a function. But I don't think mentioning it as being keyword is useful either - I think referring it as a "for range" loop or "range form of the for loop" is descriptive enough.

**Comment by andrerfcsantos** [comment]
Hi!

Thanks for the review. I followed all the suggestions expect the one for the `range-iteration`, which I commented.

I also added a blurb for the "comments" concept which I noticed was added meanwhile.

File: concepts/range-iteration/.meta/config.json (hunk 6 above)
Comment (jmrunkle) [reply concepts/range-iteration/.meta/config.json:2]: SG, its behavior just acts like a function from my perspective. Resolving this comment sicne this works.

**Review by jmrunkle — APPROVED** [review]

**Comment by jmrunkle** [comment]
I am pretty sure that failing test is just a timeout / not your fault.

**Pull request merged**